src/lib/loadsettings.cc

```diff
@@ -143,6 +143,7 @@ LoadPage::LoadPage():
 	blockLocalFileAccess(true),
 	stopSlowScripts(true),
 	debugJavascript(false),
+	debugNetworkRequests(false),
```

Comment:
I think we can simply remove this in favor of the new `debug` log level, no point in having more settings.

Reply:
sounds good, I just figured it might be "too" chatty, but if you're cool with it I'm also cool with it